src/elevensLab/Card.java

```diff
@@ -0,0 +1,35 @@
+package elevensLab;
+
+public class Card {
+	
+	private String rank;
+	private String suit;
+	private int pointValue;
+	
+	public Card(String r, String s, int pV) {
+		rank = r;
+		suit = s;
+		pointValue = pV;
+	}
+	
+	public String getRank() {
+		return rank;
+	}
+	
+	public String getSuit() {
+		return suit;
+	}
+	
+	public int getPointValue() {
+		return pointValue;
+	}
+	
+	public boolean isEqual(Card otherCard) {
+		return this.getRank() == otherCard.getRank() && this.getSuit() == otherCard.getSuit() && this.getPointValue() == otherCard.getPointValue();
```

Comment:
I got the description for the homework! I can look at it now. 
Very good implementation that you should be proud of. 
a few things:
1. I like the one-liner implementation! but remember you can press "return" and put one line in two lines so people don't need to scroll around. (It works that way, trust me)
2. for strings, you should use `String equals(String other)` instead of `==`.  I am pretty sure that `==` would not work here bc STRINGS ARE IMMUTABLE. 
3. also, inside a class, feel free to use the actual variable instead of the "caller" functions like `getSuit()` to decrease the length. You can also call directly to the `otherCard.suit` and "otherCard.rank". Magical, right? I honestly believe that the caller functions are very bureaucratic and lengthy but the AP wants you to write them.